scripts/timelapse-colorizer-data/generate_data.py

```diff
@@ -19,7 +22,7 @@
 # python timelapse-colorizer-data/generate_data.py --output_dir /allen/aics/animated-cell/Dan/fileserver/colorizer/data --dataset mama_bear
 # python timelapse-colorizer-data/generate_data.py --output_dir /allen/aics/animated-cell/Dan/fileserver/colorizer/data --dataset goldilocks
 
-# DATASET SPEC:
+# DATASET SPEC: See DATA_FORMAT.md for more details.
```

Comment:
I guess most of the lines below could be moved out of here and into data_writer_utils

Reply:
I think I'll remove this for now and continue pointing people to DATA_FORMAT.md via link! It'll be easiest if all the documentation lives in once place and doesn't have to be replicated... 😅